llvm/test/TableGen/EncodingConflict.td

```diff
@@ -0,0 +1,108 @@
+// RUN: llvm-tblgen -gen-searchable-tables -I %p/../../include %s | FileCheck %s
```

Comment:
Can you rename this test to refer to the feature being tested? Not a particular usage of the feature.

Reply:
Made it to ReturnRange.td